RallyHereIntegration/Source/RallyHereIntegration/Public/RH_LocalPlayerLoginSubsystem.h

```diff
@@ -217,6 +219,17 @@ class RALLYHEREINTEGRATION_API URH_LocalPlayerLoginSubsystem : public URH_LocalP
 	UPROPERTY(BlueprintAssignable, Category = "Login")
 	FRH_OnLoginCompleteDynamicMulticast BLUEPRINT_OnLoginComplete;
 
+	/**
+	 * @brief Multicast delegate that gets broadcasted when a player's crossplay setting is changed.
+	 */
+	FRH_GeneralSettingChangedMulticast OnCrossplaySettingChanged;
+
+	/**
+	 * @brief Multicast delegate that gets broadcasted when a player's crossplay setting is changed.
+	 */
+	UPROPERTY(BlueprintAssignable, Category = "Login")
```

Comment:
Is this supposed to be in the Login category? Seems like an odd place for a changed setting (granted I don't know the other categories)

Reply:
While I generally agree, it is because this is on the login subsystem (which is where the crossplay settings currently live, since they are involved in the login process)